scss/civicrm/common/_modals.scss

```diff
@@ -144,6 +146,7 @@
           $btn-primary-color, $btn-primary-bg, $btn-primary-border
         );
 
+        $ui-icon-margin: 7px;
```

Comment:
Not quite sure we need a var now here. I am also not sure why we have this 7px margin.

Reply:
jQuery UI adds a -8px margin-top to the icons. This variable overrides that value so it's better aligned (need to check why they are look back like -8px instead of -7px)

Reply:
@reneolivo OK, then I think this var needs to be outside and there needs to be a comment that it overrides this -7px by jQuery.